src/c++/perf_analyzer/load_manager.h

```diff
@@ -212,6 +212,18 @@ class LoadManager {
   /// Stops all the worker threads generating the request load.
   void StopWorkerThreads();
 
+  /// Wrapper method for calling InferInput::Create().
```

Comment:
lets not couple the documentation with the implementation. You can just say that this creates an InferInput and returns it in the infer_input pointer.

Reply:
Updated documentation 👍